R/highlight_function.R

```diff
@@ -0,0 +1,55 @@
+#' Highlight palette prep function
+#'
+#' Create named list to serve as palette input for following fill/color highlight functions
+#'
```

Comment:
I believe this wants a `@noRd` tag, at least according to what I've seen @nmpeterson write.